### ISSUE TITLE
Backend: "IPFS timeout" exception when creating an offer

### ISSUE THREAD

**Issue by franckc** [OriginProtocol/dshop#225]
Seen in the server logs in prod:
2020-07-15T23:14:27.214240084Z [INFO]  dshop.utils.handleLog: Event OfferCreated for listing Id 1-001-233
2020-07-15T23:14:27.535264693Z [INFO]  dshop.utils.handleLog: OfferCreated - 559 by 0xDF73aF150b8E446a6D39FDdc2CFA7Bf067B88936
2020-07-15T23:14:27.535335301Z [INFO]  dshop.utils.handleLog: IPFS Hash: QmZBXbhdAa37sAbCX1h95coAePvuXiwwjn92hbvjeXE3Xp
2020-07-15T23:14:37.539693866Z [ERROR] dshop.utils.handleLog: IPFS gateway timeout

Associated Sentry error: https://sentry.io/organizations/origin-protocol/issues/1786607561/events/de24376de8ac41cab0655d887f5beb12/

Sentry shows the stacktrace as:
```
      statusStr: 'error',
      data: { error: e.message },
      updatedBlock: event.blockNumber,
      createdAt: new Date(event.timestamp * 1000),
      createdBlock: event.blockNumber,
      ipfsHash: event.ipfsHash
    })
    Sentry.captureException(e)
    return order
  }
```

Which is here: https://github.com/OriginProtocol/dshop/blob/master/backend/utils/handleLog.js#L312

**Comment by franckc** [comment]
CC @mikeshultz to keep him in the loop since this is IPFS related.

**Comment by mikeshultz** [comment]
The bull job appears to have been completed.  Not sure if that's also an error or an indication that it completed on a retry.  Apparently the UI we have doesn't provide us with that info?

**Comment by franckc** [comment]
So I'm not sure why but the code does not throw. It records the error in the DB and moves on.
See https://github.com/OriginProtocol/dshop/blob/master/backend/utils/handleLog.js#L313
It should probably throw now that we have a retryable queue.

My guess is that the timeout come from fetching the offer data on IPFS.
This line: https://github.com/OriginProtocol/dshop/blob/master/backend/utils/handleLog.js#L244

**Comment by mikeshultz** [comment]
That was my guess too.  The [offer data is up](https://fs.ogn.app/ipfs/QmZBXbhdAa37sAbCX1h95coAePvuXiwwjn92hbvjeXE3Xp).  IPFS p2p probably just didn't resolve it fast enough.  It would've worked on subsequent attempts if it didn't fake-succeed.

**Comment by franckc** [comment]
Should be fixed by #269
